FE/todo-app/src/components/formComponents/Error.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ComponentPropsWithoutRef, FC } from "react";
+import { cn } from "@/lib/utils";
+
+type Props = ComponentPropsWithoutRef<"p"> & {
+  errorMsg?: string;
+  className?: string;
+  ariaLabel?: string;
+  ariaLive?: "off" | "polite" | "assertive";
+  ariaAtomic?: boolean;
+};
+
+export const Error: FC<Props> = (props: Props) => {
+  const { errorMsg, className, ariaLabel, ariaLive, ariaAtomic, ...rest } = props;
```

Comment:
stejný problém jak v jiných komponentách... prosp:Prosp když máš Props v generice FC a desktrukture udělej přímo v argumetech funkce

Reply:
Tohle si zkontroluj ve všech komponentách, nebudu ti to psát všude dokola :)